src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue

```diff
@@ -322,17 +324,19 @@ export default {
         data: this.raw['data'],
         from: this.raw['from'],
         to: this.raw['to'],
-        value: this.raw['value'],
+        value: !this.$route.fullPath.includes('reveal') ? this.raw['value'] : 0,
```

Comment:
I really dont like this way of checking `path.includes` shouldn't the value be 0 if it is reveal anyway?